lib/src/modules/menuItem/menu-item.scss

```diff
@@ -0,0 +1,29 @@
+.menu-item {
+  list-style: none;
+  a {
+    cursor: pointer;
+  }
+
+  &.disabled {
+    pointer-events: none;
+  }
+}
+
+.dropdown__content .menu-item > a {
+  @apply text-neutral-20 text-base py-3 px-4 block;
+  text-decoration: none;
+  white-space: nowrap;
+  &:hover, &:focus {
+    @apply text-neutral-20 bg-neutral-98;
+  }
+}
+
+.dropdown__content .menu-item.disabled {
+  opacity: 0.5;
```

Comment:
🟢  a very minor thing but what is a menu item isn't in a dropdown?

Should it not use these styles?

Reply:
Ah we use it for navigation links mainly - its just sometimes used in dropdowns